studiengaenge/lehramt_allgemein.tex

```diff
@@ -3,25 +3,33 @@ \section{Die 50\%-Bachelor-Studiengänge (Lehramt)} % Dereinst "Lehramt allgemei
 
-Bisher ist vieles noch nicht endgültige geklärt und einiges noch völlig unklar. Wenn sich etwas Neues ergibt, werden wir dies euch natürlich im FS-Info\footnote{Eine Mailing Liste, in der wir jeden Monat über die wichtigsten Dinge informieren. Anmeldung dafür unter \url{https://mathphys.fsk.uni-heidelberg.de/mailman/listinfo/fsinfo}} oder in den News unserer Webseite\footnote{\url{https://mathphys.fsk.uni-heidelberg.de/w/}} mitteilen.
+\subsection{praktische Tipps für Lehrämtler}
+In einem Lehramtsstudium wird es sicherlich auch vorkommen, dass ihr Veranstaltungen in der Altstadt besuchen werdet, vor allem wenn sich euer Zweitfach in der Altstadt befindet. Deshalb solltet ihr beim Zusammenstellen eures Stundenplans darauf achten, genügend Zeit einzuplanen um rechtzeitig in der Altstadt (oder im Feld) anzukommen.
+
+Es gibt prinzipiell zwei Möglichkeiten um zwischen dem Neuenheimer Feld und der Altstadt zu pendeln. Zum einen könntet ihr mit dem Bus Nr. 31/32  fahren zum anderen mit dem Fahrrad. In beiden Fällen ist es zu empfehlen mit eine halbe Stunde von Hörsaal zu Hörsaal zurechnen. Allerdings ist das Fahrrad eindeutig das zuverlässigere Verkehrsmittel. Mehr dazu findet ihr unter dem Abschnitt Verkehrsmittel in Heidelberg. 
+Ein weiterer Tipp ist, dass ihr euch genügend Zeit zum Zettelrechnen in den Stundenplan einplant, da man in jeder Mathematik-, Informatik- und Physikvorlesung Zettel erhält. Gemeinsam rechnen sich die Zettel deutlich besser. 
+
+Außerdem wird es vorkommen, dass sich gerade bei fakultätsfremden Fächern einzelne Veranstaltungen überschneiden. Genauso kann es vorkommen, dass der Arbeitsaufwand der vorgesehenen Module der beiden Fächer in einem Semester kaum zu leisten ist. Dann solltet ihr keine Scheu haben den Stundenplan nicht vollständig nach dem Studienverlaufsplan zu gestalten. Sollten dabei Probleme auftauchen, könnt ihr euch Hilfe bei den jeweiligen Fachstudienberatern suchen.
+
+%Bisher ist vieles noch nicht endgültige geklärt und einiges noch völlig unklar. Wenn sich etwas Neues ergibt, werden wir dies euch natürlich im FS-Info\footnote{Eine Mailing Liste, in der wir jeden Monat über die wichtigsten Dinge informieren. Anmeldung dafür unter \url{https://mathphys.fsk.uni-heidelberg.de/mailman/listinfo/fsinfo}} oder in den News unserer Webseite\footnote{\url{https://mathphys.fsk.uni-heidelberg.de/w/}} mitteilen.
```

Comment:
Da wir weiter unten auf den Master of Education noch eingehen, was eigentlich die einzige große Baustelle noch ist, würde ich die Zeile einfach rausschmeißen.